### PR TITLE
Project completion Changes from Dev

### PR DESCRIPTION
**Pull request by xarrijorge**
Entire site build from the dev branch. This the full project with all files, edits and checks completed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+        <p>
+        Dr. Kovac has spent his career trying to read the secrets of these waves. 
+        He is one of four leaders of Bicep, which has operated a series of 
+        increasingly sensitive radio telescopes at the South Pole, where the thin, 
+        dry air creates ideal observing conditions. The others are Clement Pryke of the University of Minnesota, 
+        Jamie Bock of the California Institute of Technology and Dr. Kuo of Stanford.
+        </p>
+      </div>
+      <!-- Google Ads -->
+      <div class="ads">
```
Comment (azdrenymeri) [inline index.html]: This one remove it

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+    margin: 0;
+
+}
+
+
+/* Navbar */
+nav{
+    display: flex;
+    position:sticky;
+    position: -webkit-sticky;
```
Comment (azdrenymeri) [inline assets/css/style.css]: CSS property `position: -webkit-sticky` is invalid I did a research on this and in official docs on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/position) they use it so its a little bit wierd to be an invalid value in both browsers chrome and firefox but anyway make sure to remove it and leave the standard way without `-webkit-`.

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+    align-items: center;
+    border-bottom: 1px solid gray;
+    background: #fff;
+    z-index: 5000;
+    margin-bottom: 20px;
+    top: 0;
+}
+nav img{
+  width: 250px;
+  /* height: 40px; */
```
Comment (azdrenymeri) [inline assets/css/style.css]: remove this commented property

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+  font-family: sans-serif;
+}
+a:hover{
+  text-decoration: underline;
+}
+button{
+  background: #6288a5;;
+  min-height: 5vh;
+  border-radius: 3px;
+  border: #6288a5;
```
Comment (azdrenymeri) [inline assets/css/style.css]: `border` property is a shorthand for 3 other properties `border-style`, `border-color`, `border-width` and you have given only one value.

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+  color: black;
+  font-weight: 200;
+  line-height: 0;
+}
+.read-comments{
+  text-transform: uppercase;
+}
+
+.article-image-video{
+  widows: 560px;
```
Comment (azdrenymeri) [inline assets/css/style.css]: `widows` property accepts only integer values and global values without any unit.

File: assets/css/style.css [hunk 6]
```diff
@@ -0,0 +1,383 @@
...
+  height:315;
```
Comment (azdrenymeri) [inline assets/css/style.css]: you forgot to add a unit at the end of `height` property.

File: assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,383 @@
...
+  padding: 0.5em;
+  font-size: 9px;
+}
+.footer-copyright a{
+    color: #909090;
+    padding: 0;
+}
+.fas, .fa, .fab{
+  border-radius: 50%;
+  background: #e6e6e6;
```
Comment (azdrenymeri) [inline assets/css/style.css]: Same here `border` property accepts at least 2 values

File: assets/css/style.css [hunk 8]
```diff
@@ -0,0 +1,383 @@
...
+  width: 30px;
+  height: 30px;
+  border:#e6e6e6;
+  text-align: center;
+  padding-top: 8px;
+  font-size: 12px;
+}
+.fa:hover, .fas:hover,.fab:hover{
+  background: #fff;
```
Comment (azdrenymeri) [inline assets/css/style.css]: Same as line [372](https://github.com/xarrijorge/nytimes_article/pull/2/files#diff-b8392010d5f6b5c8a2f493b3a11e3109R372) and [72](https://github.com/xarrijorge/nytimes_article/pull/2/files#diff-b8392010d5f6b5c8a2f493b3a11e3109R72)

File: assets/css/style.css (hunk 4 above)
Comment (xarrijorge) [reply assets/css/style.css]: border-color, border-width and border-style properties are also shorthand for setting individual styles for top, right, bottom and left borders. So what is your advice?

**Pull request closed without merging**